content/300-guides/300-upgrade-guides/200-upgrading-versions/030-upgrading-to-prisma-4.mdx

```diff
@@ -52,9 +52,7 @@ This section includes changes that affect both the Prisma Schema and the Prisma
 
 #### Node.js minimum version change
 
-From Prisma version 4.0.0, the minimum version of Node.js that we support is 14.17.x. If you use an earlier version of Node.js, you will need to update it.
```

Comment:
Yes. There's also another doc that I forgot to change - I'll update it a bit later.

Reply:
Oh sorry if I triggered that, I really just meant in the `content/300-guides/300-upgrade-guides/800-upgrade-from-prisma-1/08-upgrade-from-mongodb-beta.mdx` (Edit below, so that one is fine). Here the specific version that this was updated to in Prisma 4.0.0 is ok to mention and would stay unchanged in future major versions.

Reply:
Yeah, I was in two minds about whether to apply it here. In the end I went with docs maintainability. However, I doubt we would let version numbers go stale, because when they change, we can do a global find in the docs source files.

Reply:
The version number that was made a requirement for Prisma 4 will not change, that is my point. We are also not adding all the additional P4 upgrade steps to the P3 upgrade guide etc as that really just describes the step from 2.x to 3.x. And this one here describe from 2.x or 3.x to 4.x - and for that the requirement will always and forever be `14.17.x` as we will only be able to change that requirement to a higher version with Prisma 5.x.

Reply:
I've reverted this change in PR [3374](https://github.com/prisma/docs/pull/3374)